README.rst

```diff
@@ -72,6 +70,13 @@ and puts them into "objects" field of the CDR item according to CDR v3 schema.
         objects=['http://example.com/1.png', 'http://example.com/1.png'],
     )
 
+4. Optionally, subclass and re-define:
+
+   - ``media_request`` method if you want to
+     customize how media items are downloaded.
+   - ``s3_path`` method if you want to customize S3 URL of stored items.
+     By default it generates https:// url for public items and s3:// for private.
```

Comment:
It'd be nice to expand a bit on this: how is it determined if an item is public/private, and what is Scrapy default.